git-submodule-bump

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+# Convenient script to create commit for update of submodules.
+#
+# NOTE: This script can only handle one submodule update. In the spirit of
+# keeping commits atomic, we typically shouldn't update multiple submodules in
+# the same commit anyway.
+#
+# Author: Zeeshan Ali <zeeshan.ali@pelagicore.com>
+
+SHORTLOG=${1}
+if test -z ${SHORTLOG}; then
+    echo -e "Usage:\n"
+    echo "${0} COMMIT_SUMMARY"
+    exit -1
+fi
+
+CHANGED_MOD=$(git submodule status | grep '^+' | tr -d + | head -n 1)
+if test -z ${CHANGED_MOD}; then
```

Comment:
Please see http://unix.stackexchange.com/questions/86800/quote-a-variable-in-shell-script-if-statement and I got the variable error depended on system.